### ISSUE TITLE
DD3 R14 Improve the per-element guidance in a number of ways: Title

### ISSUE THREAD

**Issue by PeterParslow** [agiorguk/gemini#48]
Add Schema.org equivalent element of “name” and DCAT2 equivalent of “dct:title”, as described in D3 R1.
Consider adjusting the guidance to reflect the terms suggested by the “General” section on language to use.

**Comment by Sgaff** [comment]
Is this in the 'Corresponding elements in other standards' part of each element guidance?

**Comment by PeterParslow** [comment]
Yes, an extra "other standard" entries, one for Schema.org, one for DCAT2
